test/e2e_node/jenkins/image-config-serial.yaml

```diff
@@ -5,8 +5,8 @@ images:
   ubuntu:
     image: ubuntu-gke-1604-xenial-v20170420-1 # docker 1.12.6
     project: ubuntu-os-gke-cloud
-  coreos-alpha:
-    image: coreos-alpha-1122-0-0-v20160727 # docker 1.11.2
+  coreos-stable:
+    image: coreos-stable # docker 1.12.6
```

Comment:
Image uses an image family because Container Linux removes old, unsupported, image tags. See background here: https://github.com/kubernetes/kubernetes/pull/46467#issuecomment-304144064